src/providers/fault-store/fault-store.ts

```diff
@@ -17,6 +16,8 @@ const reduxLogger = require('redux-logger');
 
 @Injectable()
 export class FaultStoreProvider {
+  summaryLookups = require('../../assets/data/fault-summary-lookup.json');
```

Comment:
This might work in development but I'm not sure if this file will be being served from this location when it is built properly? I might be wrong but can we test this?

Reply:
It seems to work okay...

Reply:
The path reference is the same whether you're in `/src/providers/fault-store/` or in `/www/build` maybe that's why it works 😕
<img width="307" alt="screen shot 2018-05-08 at 14 07 53" src="https://user-images.githubusercontent.com/33695049/39758958-45cb30ae-52c9-11e8-804a-99ae44499d05.png">
<img width="299" alt="screen shot 2018-05-08 at 14 07 15" src="https://user-images.githubusercontent.com/33695049/39758963-4809a53a-52c9-11e8-863d-6b1304b85e4c.png">